projects/angular/components/ui-grid/src/ui-grid.component.ts

```diff
@@ -1223,8 +1239,24 @@ export class UiGridComponent<T extends IGridDataEntry>
     }
 
     isFilterApplied(column: UiGridColumnDirective<T>) {
-        return (column.dropdown?.value != null && column.dropdown!.value!.value !== column.dropdown!.emptyStateValue)
-            || (column.searchableDropdown?.value != null && (column.searchableDropdown?.value as ISuggestValue[])?.length !== 0);
+        const searchableHasValue = column.searchableDropdown?.value != null &&
+            (column.searchableDropdown.value as []).length !== 0;
```

Comment:
```suggestion
            (column.searchableDropdown.value as []).length > 0;
```

since length can't be lower than 0

Reply:
i was covering 2 cases but in an implicit manner (probably not the best approach).
`column.searchableDropdown.value` can be `undefined` or `ISuggestValue` or `ISuggestValue []`. So something of `ISuggestValue` would be true but an empty array would be false.

Added an explicit condition for multiple: `(!column.searchableDropdown.multiple || (column.searchableDropdown.value as []).length > 0)`